src/query/sql/src/planner/plans/insert.rs

```diff
@@ -31,6 +32,8 @@ pub enum InsertInputSource {
     StreamingWithFileFormat(FileFormatOptions, usize, Option<Arc<InputContext>>),
     // From cloned String and format
     Values(String),
+    // From sideload stage or uri
+    Sideload(Arc<SideloadOptions>),
```

Comment:
How about calls this new source as `StagedValues`? We can get about the data comes from stage by its name

Reply:
updated